tutorials/en_us/JARS/OPTIMIZATION.md

```diff
@@ -2,11 +2,73 @@
 
 Here you will learn how to optimize you Minecraft server to get better performance.
 
+## Different kinds of Lag
+
+TPS - Server Lag
+**TPS** stands for **T**icks **P**er **S**econd. It is also the only lag type a server owner has direct control over.
+
+A server processes all tasks at a rate of 20 TPS. Tasks like mob movement, crop growth, and player interactions with blocks need to be ticked by the server to function properly. A TPS below 20 means the server is running behind and must skip tasks in order to keep important tasks on time. Significant TPS loss usually presents itself with minor annoyances like intermittent mob freezing and block break resets. Severe cases could result in server-wide freezes or even crashes.
+
+**TPS Ratings**
+- 20.0 = Flawless - Well done.
+- 19.95-19.99 = Great - Unnoticeable TPS loss. Most servers live here.
+- 18.5-19.94 = Fair - Maybe some annoyances, but nothing game-ruining.
+- 16.0-18.4 = Poor - You definitely need to fix this if this is your average.
```

Comment:
Maybe say "fix it if this" - "this if this" sounds weird I think

Reply:
changed!